wandb/sdk/artifacts/artifact.py

```diff
@@ -200,6 +169,9 @@ def __init__(
         self._type: str = type
         self._description: Optional[str] = description
         self._metadata: dict = self._normalize_metadata(metadata)
+        self._ttl_duration_seconds: Optional[int] = None
+        self._ttl_is_inherited: Optional[bool] = None
+        self._ttl_change: Optional[ArtifactTTLChange] = None
```

Comment:
Since we're back to two flags, this can just be `self._ttl_changed: bool = False`.